apps/fishing-map-e2e/cloudbuild.yaml

```diff
@@ -60,6 +60,7 @@ steps:
     waitFor: ['restore_cypress_cache']
     entrypoint: 'bash'
     env:
+      - 'CYPRESS_publicApiGateway=$_NEXT_PUBLIC_API_GATEWAY'
```

Comment:
As we are using the `NEXT_PUBLIC_API_GATEWAY` env variable is [this line](https://github.com/GlobalFishingWatch/frontend/pull/2170/files#diff-308681de6a66133e880aabfcbdc55961cdc5ca61b838a4b36d90bba82e523c09R10) used?